### DISCUSSION TITLE
How to manually update HA to latest node-zwave-js version?

### DISCUSSION THREAD

**Discussion by bluemantwo** [zwave-js/zwave-js#5842] [General] [answered]
Every few weeks or so, there is an update I see available in Home Assistant for Zwave that allows me to get the latest node-zwave-js.  Currently, that puts me on 10.21.0.  But if I want to update to the very latest version here (currently 10.22.2), how do I do that in Home Assistant?

**Comment by kpine** [comment, marked as answer]
That's not something you can do. The application updates will include new driver versions at the maintainers' discretion.

Well, if you want to build your own software, you can.

**Comment by bluemantwo** [comment]
Thanks for the reply.  I was just curious.  

Cheers.
